ql/src/java/org/apache/hadoop/hive/ql/QueryPlan.java

```diff
@@ -121,6 +121,8 @@
   private final DDLDescWithWriteId acidDdlDesc;
   private Boolean autoCommitValue;
 
+  private Boolean prepareQuery;
```

Comment:
It seems this should be a boolean given the return type of the methods getter/setter.